extensions/ql-vscode/src/extension.ts

```diff
@@ -775,6 +774,14 @@ async function activateWithInstalledDistribution(
       cancellable: true
     })
   );
+
+  ctx.subscriptions.push(
+    commandRunner('codeQL.monitorRemoteQuery', async (
+      query: RemoteQuery,
+      token: CancellationToken) => {
+      await rqm.monitorRemoteQuery(query, token);
+    }));
```

Comment:
You can remove the progress monitor and the cancellation token.

```suggestion
    commandRunner('codeQL.monitorRemoteQuery', async (
      query: RemoteQuery) => {
      await rqm.monitorRemoteQuery(query, token);
    }));
```

Reply:
Hmm but I'm using the token to allow cancellation. What am I missing?

Reply:
Oops...I missed that. Then you need to use `commandRunnerWithProgress` and just ignore the `progress` parameter.

Reply:
Are we sure we need `commandRunnerWithProgress` here? If I use that I get a persistent notification (see below), which I don't think it matches the expected UX.

![image](https://user-images.githubusercontent.com/311693/145042457-29292eba-f3bb-4b88-a131-5008c34a76d7.png)